packages/components/src/focal-point-picker/utils.js

```diff
@@ -1,10 +1,6 @@
 export const INITIAL_BOUNDS = {
-	top: 0,
-	left: 0,
-	bottom: 0,
-	right: 0,
-	width: 0,
-	height: 0,
+	width: 200,
+	height: 170,
 };
```

Comment:
The width and height values are from the styles and made the updated the styles pull them from here. The component was also no longer in need of all the other properties.